src/bijectors/truncated.jl

```diff
@@ -68,6 +68,27 @@ end
 
 with_logabsdet_jacobian(b::TruncatedBijector, x) = transform(b, x), logabsdetjac(b, x)
 
+# from stan https://github.com/stan-dev/math/blob/develop/stan/math/prim/constraint/lub_constrain.hpp etc
```

Comment:
Is this actually necessary (or helpful)? It's simply the logarithm of the absolute value of the (arguably straightforward) derivative of `truncated_invlink` above.
```suggestion
```